docker-compose.yml

```diff
@@ -1,10 +1,13 @@
 version: '3.5'
+
 services:
     dcsa-ebl:
         build:
            .
         ports:
             - 9090:9090
+        environment:
+            - "SPRING_PROFILES_ACTIVE=local"
```

Comment:
I am not sure this would be optimal to enable by default (none of the other applications have a custom profile "hard-coded" into their compose file and I like the consistency here).  Could you perhaps move it to a `.env` file?

Reply:
Removed